www/js/app.js

```diff
@@ -43,7 +44,8 @@ locales.forEach(locale => {
   }
 });
 
-i18next.init({
+i18next.use(initReactI18next)
```

Comment:
@Abby-Wheelis
Note that our use of i18next in React is dependent on the changes here in `app.js` - I also included this in https://github.com/e-mission/e-mission-phone/pull/990 and I believe @shankari plans to merge that soon

Reply:
This has now been merged